baseapp-auth/testproject/settings.py

```diff
@@ -36,6 +46,7 @@
 AUTHENTICATION_BACKENDS = [
     "django.contrib.auth.backends.ModelBackend",
     "baseapp_auth.permissions.UsersPermissionsBackend",
+    "allauth.account.auth_backends.AuthenticationBackend",
```

Comment:
Double check this AI review: https://github.com/silverlogic/baseapp-backend/pull/185/files#r1849105895